wire-library/wire-runtime/src/jvmMain/kotlin/com/squareup/wire/DurationProtoAdapter.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire
+
+import com.squareup.wire.FieldEncoding.LENGTH_DELIMITED
+import java.time.Duration
+
+internal object DurationProtoAdapter : ProtoAdapter<Duration>(
```

Comment:
Why does this exist if there's one in common?

Reply:
left over we forgot to delete. Fixed.